demo/tsconfig.json

```diff
@@ -23,6 +23,7 @@
         "baseUrl": ".",
         "paths": {
             "*": [
+                "./node_modules/tns-core-modules/*",
```

Comment:
AFAIK that's fixed by adding `skipLibCheck: true` to tsconfig.json (what this PR does also). I had that same issue and adding that property made webpack behave nicely.